ansible/roles/hypha/templates/wtsh-hypha-web.service.jinja2

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=Hypha web docker container
+After=docker.service
+Wants=network-online.target docker.socket
+Requires=docker.socket
+
+[Service]
+Restart=on-failure
+ExecStart=/usr/bin/docker run \
+    --name wtsh-hypha-web \
+    --pull never \
+    --rm \
+    --env-file /opt/hypha/wtsh-hypha-secrets.env \
+    --env PYTHONDONTWRITEBYTECODE=1 \
+    --env PYTHONUNBUFFERED=1 \
+    --env PROJECTS_ENABLED=1 \
+    --env ORG_SHORT_NAME=WT.SH \
+    --env ALLOWED_HOSTS={{ hypha_domain }} \
+    --env SECURE_SSL_REDIRECT=0 \
+    --env DATABASE_URL=postgres://hypha:hypha@wtsh-hypha-db/hypha \
+    --network wtsh-hypha \
+    --publish 127.0.0.1:9001:9001 \
+    ghcr.io/torchbox/wtsh-hypha:{{ hypha_docker_tag }}
```

Comment:
I think this would also need a volume to persist uploaded files (`MEDIA_ROOT`) 🤔